ci/release.sh

```diff
@@ -37,6 +37,11 @@ if [[ "${BUILD_SOURCEBRANCHNAME:-}" == "master" ]]; then
     ./bazel-bin/release/release --artifacts release/artifacts.yaml --upload --log-level debug --release-dir "${release_dir}"
 else
     step "run release script (dry run)"
-    ./bazel-bin/release/release --artifacts release/artifacts.yaml --log-level debug --release-dir "${release_dir}"
+    # For use in testing the release procedure.
+    #export GPG_KEY="** ENTER KEY HERE **"
+    #export MAVEN_URL="https://oss.sonatype.org"
+    #export MAVEN_USER="** ENTER SONATYPE JIRA USERNAME HERE **"
+    #export MAVEN_PASSWORD="** ENTER SONATYPE JIR PASSWORD OR USER TOKEN HERE **"
+    ./bazel-bin/release/release --artifacts release/artifacts.yaml --log-level debug --release-dir "${release_dir}" --all-artifacts
```

Comment:
Why `--all-artifacts`? I don’t see how uploading to maven changes whether artifacts are platform dependent or not.

Reply:
Darn, another thing that slipped in.

Reply:
Fixed in 7a8baf12b